src/Products/ZCatalog/tests/test_catalog.py

```diff
@@ -145,7 +145,7 @@ def test_add_field_index(self):
         catalog = self._make_one()
         idx = FieldIndex('id')
         catalog.addIndex('id', idx)
-        self.assert_(isinstance(catalog.indexes['id'], FieldIndex))
+        self.assertTrue(isinstance(catalog.indexes['id'], FieldIndex))
```

Comment:
This line could also use `assertIsInstance`.

Reply:
+1